### PR TITLE
Added checkstyle and fixed its issues

### LINKED ISSUE #19: Add checkstyle validation
We need to add checkstyle validation to the codebase.

### PR DESCRIPTION
**Pull request by alparslanavci**
Fixes #19

### CODE REVIEW

File: src/main/java/com/hazelcast/commandline/member/names/package-info.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This package contains the class that generates Moby names
```
Comment (mesutcelik) [inline src/main/java/com/hazelcast/commandline/member/names/package-info.java]: good to give a reference about what Moby names mean....

File: src/main/java/com/hazelcast/commandline/package-info.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This package contains the classes related to main command
```
Comment (mesutcelik) [inline src/main/java/com/hazelcast/commandline/package-info.java]: what is main command?

File: src/main/java/com/hazelcast/commandline/member/MemberCommandLine.java [hunk 3]
```diff
@@ -162,9 +195,12 @@ public void list() {
     }
 
     @Command(description = "Display the logs for Hazelcast member with the given ID.", mixinStandardHelpOptions = true)
-    public void logs(
-            @Parameters(index = "0", paramLabel = "<name>", description = "Unique name of the process to show the logs, for ex.: brave_frog.") String name,
-            @Option(names = {"-n", "--numberOfLines"}, paramLabel = "<lineCount>", description = "Display the specified number of lines (default: 10).", defaultValue = "10") int numberOfLines)
+    public void logs(@Parameters(index = "0", paramLabel = "<name>",
+                                 description = "Unique name of the process to show the logs, for ex"
```
Comment (mesutcelik) [inline src/main/java/com/hazelcast/commandline/member/MemberCommandLine.java]: That looks like we make it less readable by making description multiline. Can we put such things into one line?

File: src/main/java/com/hazelcast/commandline/Verbosity.java [hunk 4]
```diff
@@ -1,7 +1,26 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package com.hazelcast.commandline;
 
 import picocli.CommandLine.Option;
 
+/**
+ * Verbosity class
```
Comment (mesutcelik) [inline src/main/java/com/hazelcast/commandline/Verbosity.java]: What is Verbosity Class? Maybe some reference to what it is?

File: pom.xml [hunk 5]
```diff
@@ -73,4 +74,41 @@
...
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-checkstyle-plugin</artifactId>
+                        <version>${maven.checkstyle.plugin.version}</version>
+                        <executions>
+                            <execution>
+                                <phase>validate</phase>
+                                <goals>
+                                    <goal>checkstyle</goal>
```
Comment (mesutcelik) [inline pom.xml]: Can you update readme how to locally validate checkstype? `mvn clean validate` I guess before the compilation. That is mostly asked by community when they hit some checkstyle issues.

File: src/main/java/com/hazelcast/commandline/member/MemberCommandLine.java (hunk 3 above)
Comment (alparslanavci) [reply src/main/java/com/hazelcast/commandline/member/MemberCommandLine.java]: Done 👍

File: src/main/java/com/hazelcast/commandline/member/names/package-info.java (hunk 1 above)
Comment (alparslanavci) [reply src/main/java/com/hazelcast/commandline/member/names/package-info.java]: Done 👍

File: src/main/java/com/hazelcast/commandline/package-info.java (hunk 2 above)
Comment (alparslanavci) [reply src/main/java/com/hazelcast/commandline/package-info.java]: I changed it to `base hazelcast command`.

**Pull request merged**